### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by exacs**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,10 @@
   "scripts": {
     "start": "NODE_ENV=production nodemon app.js",
     "start-dev": "cross-env NODE_ENV=development nodemon app.js",
-    "test": "npx tape \"test/unit/**/*.js\" | tap-spec"
+    "test": "npx tape \"test/unit/**/*.js\" | tap-spec",
```
Comment (Delodax) [inline package.json]: npx is installed with a "vanilla" installation of node or npm?

File: package.json [hunk 2]
```diff
@@ -7,7 +7,10 @@
...
+    "test:integration": "cross-env NODE_ENV=development tape \"test/integration/**/*.js\" | tap-spec",
+    "test:docker-unit": "apk update; apk add python make; npm install --no-optional; npm test",
```
Comment (Delodax) [inline package.json]: - apk is package manager in Docker image?
- Why do we need python?

File: test/integration/createCoursesFile.test.js [hunk 3]
```diff
@@ -1,10 +1,18 @@
-var test = require('tape')
+require('dotenv').config()
+const test = require('tape')
 const fs = require('fs-extra')
 
 const {syncCoursesSectionsAndEnrollments} = require('../../run-periodically')
-test.skip('should create the file with correct name, headers, and a line including a sisCourseId', async t => {
```
Comment (Delodax) [inline test/integration/createCoursesFile.test.js]: Why was this test skipped before? Temporary solution?

File: package.json [hunk 4]
```diff
@@ -7,7 +7,10 @@
   "scripts": {
     "start": "NODE_ENV=production nodemon app.js",
     "start-dev": "cross-env NODE_ENV=development nodemon app.js",
-    "test": "npx tape \"test/unit/**/*.js\" | tap-spec"
```
[the remark this answers is not in the document]
Comment (exacs) [reply package.json]: Updated to tap-spec^5.0.0

**Pull request merged**